app/views/pages/developer-guide/apis/creating-items-from-external-api-call.liquid

```diff
@@ -73,7 +73,7 @@ Here is an example POST request you need to send. You need to set:
 curl https://example.platform-os.com/api/user/models
   -H "Authorization: Token token=[YOUR API TOKEN]"
   -H 'Content-Type: application/json'
-  --data '{"form": { "properties_attributes: {"name": "Red flowers", "creator_id":"1" }}, "form_name": "import_flowers", "parent_object_class":"CustomModelType","parent_object_id":"flower"}'
+  --data '{"form": { "properties_attributes: {"name": "Red flowers", "creator_id":"1" }}, "form_name": "import_flowers", "parent_object_class":"CustomRecordType","parent_object_id":"flower"}'
```

Comment:
@diana-lakatos `CustomModelType` is the oldest name for ModelSchema and now Table, it cannot be changed. Unfortunately, this needs to stay `CustomModelType` for now

Reply:
Edited, thank you!